js/scheduler.js

```diff
@@ -199,35 +199,17 @@
 			this.toggleState('enable');
 		},
 
-		setUtcTime: function setUtcTime(d, t, offset) {
-			var date = d.split('-');
-			var time = t.split(':');
+		setUtcTime: function setUtcTime(day, time, offset) {
+			var dateArray = day.split('-');
```

Comment:
change variable name to `dateSplit` and `timeSplit` for consistency with other split results below. why not `*Array`? personal preference. `varnameArray[0]` feels redundant.